reactquiz/src/components/quiz/Results.js

```diff
@@ -1,34 +1,25 @@
-import React,{Component} from 'react';
-
-
-class Results extends Component{
-
+import React, { Component } from 'react';
 
+class Results extends Component {
 	render() {
```

Comment:
Consider adding JSDoc-style comments to the `Results` component to describe its props (`score`, `questions`) and what the component does.